account/account.go

```diff
@@ -24,15 +25,15 @@ const (
 )
 
 type Account interface {
-	PayBidTx(context.Context, node.FullNode, common.Address, *big.Int) ([]byte, error)
+	PayBidTx(context.Context, common.Address, *big.Int) ([]byte, error)
 }
 
-func New(config *Config) (Account, error) {
+func New(config *Config, fullNode node.FullNode) (Account, error) {
```

Comment:
The design of the object model here may have a problem, `Account` model is not good to have `fullNode` info inside it,  the RPC client can have the `fullNode` info;
 It is not a very good for an account  can do `PayBidTx` by itself; usually, an account can sign, but dont have to define what it can sign, not even say an account can send tx.

`Fullnode` interface is not a good name, becase a lightnode can aslo implement `ChainID`, `PendingNonceAt` and `Balance`